layouts/_default/baseof.html

```diff
@@ -16,15 +16,19 @@
     <link href="{{ .Site.RSSLink }}" type="application/rss+xml" rel="alternate">
 
     {{ template "_internal/google_analytics_async.html" . }}
-  </head>
+
+    {{ if .Site.Params.cookie_consent_info_url }}
+      {{ partial "cookie_consent.html" . }}
+    {{ end }}
+</head>
 
 <body>
 
   <div class="blog-masthead">
     <div class="container">
       <nav class="nav blog-nav">
         <a class="nav-link {{ if .IsHome }}active{{ end }}" href="{{ .Site.BaseURL | absLangURL }}">{{ i18n "home" }}</a>
-        {{ range where .Site.Pages "Type" "!=" "post" }}
+        {{ range where .Site.Pages "Type" "==" "page" }}
```

Comment:
I think I used `!= post` on purpose at the time when I wrote it. Not sure which I like better, but anyways, it's not relevant for this particular pull request / change set, so you can leave that out.